services/common/src/redux/selectors/permitSelectors.spec.ts

```diff
@@ -0,0 +1,64 @@
+import {
+  formatPermit,
+  getEditingConditionFlag,
+  getEditingPreambleFlag,
+  getLatestAmendmentByPermitGuid,
+  getPermitByGuid,
+} from "@mds/common/redux/selectors/permitSelectors";
+import { permitReducer } from "@mds/common/redux/reducers/permitReducer";
+import {
+  storeEditingConditionFlag,
+  storeEditingPreambleFlag,
+} from "@mds/common/redux/actions/permitActions";
+import { PERMITS } from "@mds/common/constants/reducerTypes";
+import * as MOCK from "@mds/common/tests/mocks/dataMocks";
+import { RootState } from "../rootState";
+
+const mockFlagsResponse = false;
+
+const mockState = {
+  editingConditionFlag: false,
+  editingPreambleFlag: false,
+};
+
+const mockPermits = MOCK.PERMITS;
+describe("permitSelectors", () => {
+  const { editingConditionFlag, editingPreambleFlag } = mockState;
+
+  it("`getEditingConditionFlag` calls `permitReducer.getEditingConditionFlag`", () => {
+    const storeAction = storeEditingConditionFlag(mockFlagsResponse);
+    const storeState = permitReducer({} as any, storeAction);
+    const localMockState = {
+      [PERMITS]: storeState,
+    };
+    expect(getEditingConditionFlag(localMockState as RootState)).toEqual(editingConditionFlag);
+  });
+
+  it("`getEditingPreambleFlag` calls `permitReducer.getEditingPreambleFlag`", () => {
+    const storeAction = storeEditingPreambleFlag(mockFlagsResponse);
+    const storeState = permitReducer({} as any, storeAction);
+    const localMockState = {
+      [PERMITS]: storeState,
+    };
+    expect(getEditingPreambleFlag(localMockState as RootState)).toEqual(editingPreambleFlag);
+  });
+  it("getPermitByGuid returns the correct permit", () => {
```

Comment:
Added these two tests! The others are pre-existing. Whole file was moved to common + TS-ed.